app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_result = []
+
+    for peopl in people:
+        list_result.append(Person(peopl["name"], peopl["age"]))
```

Comment:
You could perform a list comprehension instead of for loop. Besides that, `peopl` is not a good variable name